envoy/config/health_checker/redis/v2/redis.proto

```diff
@@ -3,7 +3,6 @@ syntax = "proto3";
 package envoy.config.health_checker.redis.v2;
 option go_package = "v2";
 
-// [#not-implemented-hide:]
 // [#protodoc-title: Redis]
```

Comment:
Does this show in the docs? I think you might need to add this page? Also, can we link to from the custom health checker docs?

Reply:
@mattklein123 true. Just realized it is not there. Will add it.